test/conformance/enqueue/urEnqueueMemImageCopy.cpp

```diff
@@ -2,3 +2,243 @@
 // Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
 // See LICENSE.TXT
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include <uur/fixtures.h>
+
+struct urEnqueueMemImageCopyTest
+    : public uur::urQueueTestWithParam<ur_mem_type_t> {
+    // Helper type so element offset calculations work the same as pixel offsets
+    struct rgba_pixel {
+        uint32_t data[4];
+    };
+    void SetUp() override {
+        UUR_RETURN_ON_FATAL_FAILURE(urQueueTestWithParam::SetUp());
+        type = getParam();
+        size = (type == UR_MEM_TYPE_IMAGE1D) ? ur_rect_region_t{width, 1, 1}
+               : (type == UR_MEM_TYPE_IMAGE2D)
+                   ? ur_rect_region_t{width, height, 1}
+                   : ur_rect_region_t{width, height, depth};
+        buffSize = size.width * size.height * size.depth;
+        partialRegion = {
+            size.width / 2,
+            size.height > 1 ? size.height / 2 : 1,
+            size.depth > 1 ? size.depth / 2 : 1,
+        };
+
+        partialRegionOffset = {
```

Comment:
I comment describing what these two partion regions are meant to be would be nice.

Reply:
Added a couple of comments